testing/helpers/checkStyleHelper.js

```diff
@@ -0,0 +1,53 @@
+import devices from "core/devices";
+import browser from "core/utils/browser";
+
+function testInChromeOnDesktop(name, testCallback) {
+    if(devices.real().deviceType === "desktop" && browser.webkit) {
+        QUnit.testInActiveWindow.call(null, name, testCallback);
+    } else {
+        QUnit.skip.call(null, name, testCallback);
+    }
+}
+
+function checkTextOverflow(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element.parentNode);
+    assert.equal(style.textOverflow, expectedValue, "textOverflow");
+}
+
+function checkWhiteSpace(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element);
+    assert.equal(style.whiteSpace, expectedValue, "whiteSpace");
+}
+
+function checkOverflowX(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element.parentNode);
+    assert.equal(style.overflowX, expectedValue, "overflowX");
+}
+
+function checkBackgroundColor(assert, element, expectedValue) {
+    let elementBackgroundColor = window.getComputedStyle(element).backgroundColor;
+    let currentElement = element.parentNode;
+    for(let i = 0; i < 5; i++) {
```

Comment:
What does `5` mean?

Reply:
The 'for(let i = 0; i < 5; i++) {' seems to be an excessive optimization: I used '5' to break enumeration through the entire hierarchy in the case when the background color is lost.
The 'while(currentElement.parentNode != document) {' cycle requires nearly the same time and I will remove '5'.

Reply:
`while(currentElement.parentNode != document.documentElement)` I think

Reply:
Yes, it was a typo. Done.